core/src/test/scala/com/rackspace/com/papi/components/checker/wadl/WADLCheckerRaxRepStepSpec.scala

```diff
@@ -0,0 +1,1402 @@
+      val checker = builder.build (inWADL, raxRepEnabled)
+      Then ("The following assertions should hold")
+      assert(checker,"count(/chk:checker/chk:step[@type='PUSH_XML_REP']) = 6")
+      assert(checker,"count(/chk:checker/chk:step[@type='PUSH_JSON_REP']) = 6")
+      assert(checker,"count(/chk:checker/chk:step[@type='POP_REP']) = 6")
+      assert(checker, "every $s in /chk:checker/chk:step[contains(@match,'tst:')] satisfies in-scope-prefixes($s) = 'tst'")
+      assert(checker, "every $s in /chk:checker/chk:step[contains(@match,'tst:')] satisfies namespace-uri-for-prefix('tst', $s) = 'test.org'")
+      assert(checker, "every $s in /chk:checker/chk:step[contains(@match,'tst2:')] satisfies in-scope-prefixes($s) = 'tst2'")
+      assert(checker, "every $s in /chk:checker/chk:step[contains(@match,'tst2:')] satisfies namespace-uri-for-prefix('tst2', $s) = 'test.org/2'")
+      assert(checker, Start, URL("a"), Method("PUT"), ReqType("(application/xml)(;.*)?"), WellXML, XPath("/tst:some_xml"), RaxPushJSON("jsonContent","/tst:some_xml/tst:json"),
+        JsonXPath("$_?firstName"), RaxPopRep, Accept)
+      assert(checker, Start, URL("a"), Method("PUT"), ReqType("(application/xml)(;.*)?"), WellXML, XPath("/tst:some_xml"), RaxPushJSON("jsonContent2","/tst:some_xml/tst:json2"),
+        JsonXPath("$_?firstName"), RaxPopRep, Accept)
+      assert(checker, Start, URL("a"), Method("PUT"), ReqType("(application/xml)(;.*)?"), WellXML, XPath("/tst:some_xml"), RaxPushXML("xmlUser","req:header('X-XML-USER')"),
+        XPath("/tst:user"), XPath("tst:user/@firstName"), XPath("tst:user/@lastName"), RaxPopRep, Accept)
+      assert(checker, Start, URL("a"), Method("PUT"), ReqType("(application/json)(;.*)?"), WellJSON, JsonXPath("$_?firstName"), RaxPushXML("xmlContent", "$body('xml')"),
+        XPath("/tst2:user"), RaxPushJSON("/tst2:user/tst2:json"), JsonXPath("$_?lastName"), RaxPopRep, RaxPopRep, Accept)
+      assert(checker, Start, URL("a"), Method("PUT"), ReqType("(application/json)(;.*)?"), WellJSON, JsonXPath("$_?firstName"), RaxPushXML("xmlUser","req:header('X-XML-USER')"),
+        XPath("/tst:user"), XPath("tst:user/@firstName"), XPath("tst:user/@lastName"), RaxPopRep, Accept)
+      assert(checker, Start, URL("a"), Method("POST"), ReqType("(application/xml)(;.*)?"), WellXML, XPath("/tst:some_xml"), RaxPushJSON("jsonUser","req:header('X-JSON-USER')"),
+        JsonXPath("$_?firstName"), JsonXPath("$_?lastName"), RaxPopRep, Accept)
+      assert(checker, Start, URL("a"), Method("POST"), ReqType("(application/json)(;.*)?"), WellJSON, RaxPushJSON("jsonUser","req:header('X-JSON-USER')"),
+        JsonXPath("$_?firstName"), JsonXPath("$_?lastName"), RaxPopRep, Accept)
+      assert(checker, Start, URL("a"), Method("POST"), ReqType("(text/yaml)(;.*)?"), RaxPushJSON("jsonUser","req:header('X-JSON-USER')"),
+        JsonXPath("$_?firstName"), JsonXPath("$_?lastName"), RaxPopRep, Accept)
+      assert(checker, Start, URL("a"), Method("POST"
```

Comment:
Typo: Children

Reply:
I'm not exactly where the original comment was supposed to be, but the following typos were near it:
 - Childred - 2 places in 1 file.
 - Chlidren - 17 places in 3 files.